src/Components/SecondSection/ArrowButtons/ArrowButtons.js

```diff
@@ -0,0 +1,28 @@
+import styles from './ArrowButtons.module.css';
+
+function ArrowButtons() {
+  return (
+    <div className={styles.ContainerArrow}>
+      <button className={styles.button}>
+        <div className={styles.leftArrow}>
+          <div className={styles.iconLeft}>
+            <div className={styles.vector}></div>
+          </div>
+          <div className={styles.ellipse15}></div>
+          <i className={`${styles.iconArrow} fas fa-arrow-left`}></i>
+        </div>
+      </button>
+      <button className={styles.button}>
+        <div className={styles.rightArrow}>
+          <div className={styles.iconRight}>
+            <div className={styles.vector}></div>
+          </div>
+          <div className={styles.ellipse16}></div>
+          <i className={`${styles.iconArrow} fas fa-arrow-right`}></i>
+        </div>
+      </button>
```

Comment:
This is incorrect.
What if you have two more icons? How many line will your code become?
You should have an array with two objects, each object has the styling classes, then you can loop over the array and render the icons.

Reply:
Yes, I didn't notice that.
i did it .